test/suites/fips/validate-fips.robot

```diff
@@ -38,8 +41,15 @@ Teardown
 
 Microshift Binary Should Dynamically Link FIPS Ossl Module
     [Documentation]    Check if Microshift binary is FIPS compliant.
+
+    ${stdout_file}=/tmp/microshift-fips.run.log
```

Comment:
Can we generate a random file name?
```
    ${rand}=    Generate Random String
    ${local_tmp}=    Join Path    /tmp    ${rand}
```

Reply:
in theory we can, but if we are going to store this log file as CI artifact  better to have constant name.

Reply:
not according to the [man](https://man7.org/linux/man-pages/man8/ld.so.8.html):
```
       LD_DEBUG_OUTPUT (since glibc 2.1)
              By default, LD_DEBUG output is written to standard error.
              If LD_DEBUG_OUTPUT is defined, then output is written to
              the pathname specified by its value, with the suffix "."
              (dot) followed by the process ID appended to the pathname.

              LD_DEBUG_OUTPUT is ignored in secure-execution mode.
```